### DISCUSSION TITLE
Problema invio segnalazione

### DISCUSSION THREAD

**Discussion by lucabravi2** [globaleaks/globaleaks-whistleblowing-software#3949] [Q&A]
Buongiorno, ho installato l'applicazione e va tutto bene tranne che al momento nell'invio della segnalazione dopo aver compilato i campi necessari. Premetto che nella sezione notifiche, il server SMTP è correttamente configurato tant'è che le mail di notifica e di test mi arrivano.
Il problema è che se invio una segnalazione di test mi compare un errore che dice.
Errore di validazione dell'input. Unable to deliver the submission to at least one recipient.

Non so se è correlata la cosa ma mi arrivano anche via mail all'admin che dicono questo.
Globaleaks Exception

Platform: lucesulmare
Host: whistleblowing.lucesulmare.it (jmmxzdzn2q2oxe5virmuydvcisvsvrnrsa3pzmb753wofbosaswtlcid.onion)
Version: 4.14.3

OverflowError Result too large to be represented.

Traceback (most recent call last):

File "/usr/lib/python3/dist-packages/twisted/python/threadpool.py", line 244, in inContext
result = inContext.theWork() # type: ignore[attr-defined]
^^^^^^^^^^^^^^^^^^^

File "/usr/lib/python3/dist-packages/twisted/python/threadpool.py", line 260, in <lambda>
inContext.theWork = lambda: context.call( # type: ignore[attr-defined]
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

File "/usr/lib/python3/dist-packages/twisted/python/context.py", line 117, in callWithContext
return self.currentContext().callWithContext(ctx, func, *args, **kw)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

File "/usr/lib/python3/dist-packages/twisted/python/context.py", line 82, in callWithContext
return func(*args, **kw)
^^^^^^^^^^^^^^^^^

File "/usr/lib/python3/dist-packages/globaleaks/orm.py", line 185, in _wrap
result = function(session, *args, **kwargs)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

File "/usr/lib/python3/dist-packages/globaleaks/handlers/recipient/__init__.py", line 36, in get_receivertips
updated_before = datetime.fromtimestamp(int(args.get(b'updated_before', [b'32503680000'])[0]))
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

OverflowError: timestamp out of range for platform time_t

**Comment by evilaliv3** [comment]
I think you may have done a setup on a 32bit architecture.

Ensure to setup the software on a 64bit system

**Reply by lucabravi2** [reply]
Ok Thanks

**Reply by DaniloSve** [reply]
Hi Luca, could you tell us what is the correct file? I mean, in which manner can I check this option?
